tox.ini

```diff
@@ -33,6 +33,7 @@ commands = python -m towncrier.check
 [testenv:generatedocs]
 deps =
     mbed-tools-ci-scripts
+    mbed-host-tests
```

Comment:
Why in the generatedocs env? Won't the library already be installed from the `-rrequirements-test.txt` line in the root testenv?

Reply:
`mbed-tools` doesn't depend on `mbed-host-tests` anymore.